src/patch-place-break/patch-place-break-api/src/main/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/api/properties/RestrictedBlocksProperties.java

```diff
@@ -42,4 +43,17 @@ public class RestrictedBlocksProperties {
     return new RestrictedBlocksProperties(
         Collections.unmodifiableSet(new HashSet<>(materials)), mode);
   }
+
+  public boolean isRestricted(@NonNull Block block) {
+    switch (mode) {
+      case BLACKLIST:
+        return materials.contains(block.getMaterial());
+      case WHITELIST:
+        return !materials.contains(block.getMaterial());
+      case DISABLED:
+        return false;
+      default:
+        throw new IllegalStateException("Mode " + mode + " is not supported.");
```

Comment:
J'ai pris pour habitude de toujours appliquer la règle d'utiliser `String#format` quand il s'agissait de formater un string plutôt que de faire de la concaténation manuellement

```java
String.format("Mode %s is not supported.", mode)
```

C'est généralement plus clair et élégant que : 

```java
"Mode " + mode + " is not supported."
```

La valeur est d'autant plus important lorsqu'il y a plusieurs arguments attendus

Reply:
Noté merci :+1: